examples/7guis/circledraw.slint

```diff
@@ -3,21 +3,17 @@
 
 import { LineEdit, Button, Slider, StandardListView, VerticalBox } from "std-widgets.slint";
 
-struct Circle := { x: float, y: float, r: float }
+struct Circle := { x: length, y: length, d: length }
```

Comment:
Keeping the size in a a float from 0 to 1 compared to the size in number of pixel allow to resize the window and keep all the size relative.

Reply:
I think this Example should behave like drawing on a canvas. So making the window bigger makes the "canvas" bigger as well.
Alternatively we could somehow scale it with the circles, but in that case we either have to lock the x/y ratio or we get ovals when resizing.